README.md

```
@@ -13,5 +13,5 @@
 
 <div align="center">
 
-| <a href="https://github.com/DsMans0021?tab=repositories"><img align="center" src="https://github-readme-stats.vercel.app/api?username=DsMans0021&show_icons=true&theme=transparent&hide_border=true&include_all_commits=true" /></a> | <a href="https://github.com/DsMans0021?tab=repositories"><img align="center" src="https://github-readme-stats.vercel.app/api/top-langs/?username=DsMans0021&show_icons=true&theme=transparent&hide_border=true&langs_count=10&layout=compact&hide=html&exclude_repo=personal-website" /></a> |
+| <a href="https://github.com/DsMans0021?tab=repositories">https://github-readme-stats.vercel.app/api?username=Dsmans0021&show_icons=true&theme=transparent&hide_border=true</a> | <a href="https://github.com/DsMans0021?tab=repositories"><img align="center" src="https://github-readme-stats.vercel.app/api/top-langs/?username=DsMans0021&show_icons=true&theme=transparent&hide_border=true&langs_count=10&layout=compact&hide=html&exclude_repo=personal-website" /></a> |
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)